cloud/pod_util.go

```diff
@@ -107,3 +128,32 @@ func podAWSOptions(settings *evergreen.Settings) awsutil.ClientOptions {
 
 	return *opts
 }
+
+// toEnvVars converts a map of environment variables and a map of secrets to a slice of cocoa.EnvironmentVariables.
+func toEnvVars(envVars map[string]string, secrets map[string]string, secretIDs []string) []cocoa.EnvironmentVariable {
+	allEnvVars := []cocoa.EnvironmentVariable{}
+
+	for k, v := range envVars {
+		allEnvVars = append(allEnvVars, *cocoa.NewEnvironmentVariable().SetName(k).SetValue(v))
+	}
+
+	for k, v := range secrets {
+		owned := false
+
+		for _, id := range secretIDs {
```

Comment:
You don't need this because we can assume that all secrets are owned for now. We'll refactor this assumption later on when we have to share secrets across different pods.